CONTRIBUTING.md

```diff
@@ -1,7 +1,122 @@
 
-Thank you for your interest in contributing to this Rust Foundation project. 
-We are happy and excited to review and accept your pull requests.
+## Table of Contents
+
+- [Contributing to the coding guidelines](#contributing-to-the-coding-guidelines)
+  - [Table of Contents](#table-of-contents)
+  - [Contribution Workflow](#contribution-workflow)
+    - [Note on Chapter Layout](#note-on-chapter-layout)
+    - [0) Bring up the idea for discussion](#0-bring-up-the-idea-for-discussion)
+    - [1) Submit coding guideline issue](#1-submit-coding-guideline-issue)
+      - [1.1) Finding the FLS ID](#11-finding-the-fls-id)
+    - [2) Create a Draft with a Member](#2-create-a-draft-with-a-member)
+    - [3) A Pull Request is generated](#3-a-pull-request-is-generated)
+    - [4) Iterate on Feedback](#4-iterate-on-feedback)
+      - [4.1) Apply changes to the Guideline's Issue](#41-apply-changes-to-the-guidelines-issue)
+      - [4.2) Re-generate the Pull Request](#42-re-generate-the-pull-request)
+    - [5) Your Guideline gets merged](#5-your-guideline-gets-merged)
+    - [You just contributed a coding guideline!](#you-just-contributed-a-coding-guideline)
+  - [Writing a guideline locally (less typical, not recommended)](#writing-a-guideline-locally-less-typical-not-recommended)
+    - [Guideline template](#guideline-template)
+  - [Before You Begin Contributing](#before-you-begin-contributing)
+    - [Licenses](#licenses)
+    - [Code of Conduct](#code-of-conduct)
+  - [Contribution Process](#contribution-process)
+    - [Issues](#issues)
+
+## Contribution Workflow
+
+Here's a diagram of the overall process:
+
+```mermaid
+flowchart TD
+  Start(["Start"]) --> Idea["Coding Guideline Idea"]
+  Idea --> Zulip[/"(Optional)<br>0: Contributor brings <br> to discuss on Zulip"/]
+  Zulip --> CreateIssue{{"1: Contributor creates <br> issue"}}
+  CreateIssue --> Issue["Coding Guideline Issue"]
+
+  %% short local loops (no long edges)
+  S2{{"2: Review started <br> by subcommittee <br> member in <= 14 days <br><br> Contributor updates accordingly"}} --> Issue
+  Issue --> S2
+
+  Issue --> S3{{"3: Subcommitte member <br> assigns label<br>to generate PR"}} --> PR["Coding Guideline<br>Pull Request"]
+
+  S4{{"4: PR review started <br> by subcommittee member <br> in <= 14 days <br><br> Contributor discusses on PR"}} --> PR
+  PR --> S4
+
+  PR --> S5{{"5: Contributor applies <br> feedback to issue"}} --> Issue
+  Issue --> S6{{"6: Subcommittee member <br> confirms changes;<br> regenerates PR"}} --> PR
+  PR --> S7{{"7: Subcommittee member <br> approves & queues;<br>merges to main"}} --> Main[[main]]
+  Main --> End(["8: End"])
+```
+
+### Note on Chapter Layout
+
+We have the same chapter layout as the [Ferrocene Language Specification](https://spec.ferrocene.dev/) (FLS). If you would like to contribute you may find a section from the FLS of interest and then write a guideline in the corresponding chapter of these coding guidelines.
```

Comment:
Instead of "We" say the 
```suggestion
The Safety Critical Rust Coding guidelines has the same chapter layout as the [Ferrocene Language Specification](https://spec.ferrocene.dev/) (FLS). If you would like to contribute a new guideline, find a section from the FLS that interests you, then write a guideline in the corresponding chapter of these coding guidelines.
```